docs/connector-development/schema-reference.md

```diff
@@ -1,5 +1,9 @@
 # Schema Reference
 
+:::note
+You only need this if you're building a connector with Python or Java CDKs.
```

Comment:
probably good to point to the analogous page for builder and low-code. I think the closest topic is https://docs.airbyte.com/connector-development/connector-builder-ui/record-processing#declared-schema